fractions/logstash/src/main/java/org/wildfly/swarm/logstash/runtime/LogstashCustomizer.java

```diff
@@ -54,11 +57,19 @@ public void customize() {
                     .namedFormatter("logstash")
                     .properties(handlerProps);
 
+            List<String> rootLoggerHandlers = new ArrayList<>();
+
+            RootLogger existingRootLogger = this.logging.subresources().rootLogger();
+            if (existingRootLogger != null && !existingRootLogger.handlers().isEmpty()) {
+                rootLoggerHandlers.addAll(existingRootLogger.handlers());
+            }
+            rootLoggerHandlers.add(logstashHandler.getKey());
+
             this.logging
                     .customFormatter("logstash", "org.jboss.logmanager.ext", "org.jboss.logmanager.ext.formatters.LogstashFormatter",
                                      this.logstash.formatterProperties())
                     .customHandler(logstashHandler)
-                    .rootLogger(this.logstash.level(), logstashHandler.getKey());
+                    .rootLogger(this.logstash.level(), rootLoggerHandlers.toArray(new String[rootLoggerHandlers.size()]));
```

Comment:
Would you be ok making these changes?

To use the logstash level in the handler only, and then maybe using code from my comment to add handler key to root logger?